packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -512,18 +517,21 @@ class _ElevatedButtonWithIcon extends ElevatedButton {
 }
 
 class _ElevatedButtonWithIconChild extends StatelessWidget {
-  const _ElevatedButtonWithIconChild({ required this.label, required this.icon });
+  const _ElevatedButtonWithIconChild({ required this.label, required this.icon, required this.iconAlignment });
```

Comment:
```suggestion
  const _ElevatedButtonWithIconChild({
    required this.label,
    required this.icon,
    required this.iconAlignment,
  });
```

This is exceeding the `80` char limit.